private/multiset.rkt

```diff
@@ -18,6 +18,7 @@
   [multiset-unique-elements (-> multiset? immutable-set?)]
   [multiset->list (-> multiset? list?)]
   [list->multiset (-> list? multiset?)]
+  [sequence->multiset (-> sequence? multiset?)]
```

Comment:
Minor nit: the contract needs to be `(sequence/c any/c)` instead of `sequence?`, to disallow multivalue sequences like those produced by `in-hash`. The docs also have to use `(sequence/c any/c)` instead of `sequence?`.